hack/testExamplesAgainstKustomize.sh

```diff
@@ -20,7 +20,7 @@ rm -f $(go env GOPATH)/bin/kustomize
 if [ "$version" == "HEAD" ]; then
   (cd kustomize; go install .)
 else
-  GO111MODULE=on go get sigs.k8s.io/kustomize/kustomize/v3@${version}
```

Comment:
So this will need a tweak.  Suggest just passing `v3@v3.10.0` or `v4@v4.0.0`

Reply:
updated